src/GovUk.Education.ExploreEducationStatistics.Content.Model/File.cs

```diff
@@ -9,11 +8,10 @@ public class File
     {
         public Guid Id { get; set; }
 
-        [JsonIgnore]
-        public Release Release { get; set; }
-
-        public Guid ReleaseId { get; set; }
-
+        // TODO EES-1703 will reconsider the filesystem layout
+        // and potentially migrate BlobPath to be a String of the full blob path
+        public Guid BlobPath { get; set; }
```

Comment:
I'm a little bit conflicted on this. To me, `BlobPath` implies that it is a full path to the blob itself.

I'm wondering if a better name would be something like `RootPath`? At least something that tells you that this is the folder (e.g. the release directory), where anything related to this blob should be.

Reply:
I agree `RootPath` sounds a lot less confusing. I have renamed `BlobPath` to `RootPath` in a separate commit.